src/objectid.ts

```diff
@@ -7,6 +7,9 @@ import { NumberUtils } from './utils/number_utils';
 // Unique sequence for the current process (initialized on first use)
 let PROCESS_UNIQUE: Uint8Array | null = null;
 
+/** ObjectId hexString cache @internal */
+const __idCache = new WeakMap(); // TODO convert this to #__id private field when target updated to ES2022
```

Comment:
```suggestion
const __idCache = new WeakMap(); // TODO(NODE-6549): convert this to #__id private field when target updated to ES2022
```

One last change - we usually link TODOs with Jira tickets so we don't forget about them.  Can we also add a comment on the ObjectIdLike.__id with the same TODO? (I can't comment on that line because it isn't in this diff).

Reply:
Great! I was wondering if there was some process around //TODO.